### DISCUSSION TITLE
Image directive alignment CSS

### DISCUSSION THREAD

**Discussion by chrisjsewell** [pradyunsg/furo#177] [Potential Issue] [answered]
Yo, as exampled here: https://pradyunsg.me/furo/reference/images/#alignment, there's no handling for image (not figure) alignments in the CSS.

You probably already know, but `:align:` works by adding the `align-{value}` as a class.
The value can be one of `["left", "center", "right", "top", "middle", "bottom"]` (that is what it is validated against), plus sphinx adds `align-default` in places. 

Here's how docutils handles it: https://github.com/chrisjsewell/docutils/blob/8adab0660b2097b4f3c32cef7e5ff4cb3c72b084/docutils/docutils/writers/html5_polyglot/minimal.css#L181

and sphinx itself only appears to handle  left, right, center and default: https://github.com/sphinx-doc/sphinx/blob/2100722b70b1d1b024112c5ba0dbf62ad23b0154/sphinx/themes/basic/static/basic.css_t#L280

**Comment by pradyunsg** [comment, marked as answer]
Bah, Sphinx 4 broke all kinds of little things. >.<

**Reply by chrisjsewell** [reply]
😆 why's that though, are you importing some CSS from them?

**Reply by chrisjsewell** [reply]
the reason I noticed it FYI, is that I'm "borrowing" some of your SASS for: https://executablebooks.github.io/markdown-it-docutils/ 😬

**Reply by chrisjsewell** [reply]
and in-turn this will be used in the VS Code / JupyterLab extension previews 😄

**Reply by pradyunsg** [reply]
> 😆 why's that though, are you importing some CSS from them?

No, it changed the markup that the themes are supposed to stylize, in very odd + subtle ways.

Fixed this in 364daf932f09c7ea7264ff5004f468328b9e6058, and cutting a release with this now! :)

**Reply by chrisjsewell** [reply]
👍 🎉

**Reply by pradyunsg** [reply]
Heh, how our understanding of the stack evolved a few short months after this. 😅
